docs/send-data/sumo-distribution-opentelemetry.md

```diff
@@ -18,6 +19,6 @@ The Collection page provides links to download the Sumo Logic Distribution for O
 Releases are available to download in the [Sumo Logic Distribution for OpenTelemetry](https://github.com/SumoLogic/sumologic-otel-collector/releases).
 :::
 
-1. Navigate to **Manage Data > Collection** and select the **Collection** tab. On the top right of the page, click **Add Collector**.<br/>![add-collector-link.png](/img/send-data/add-collector-link.png)
+1. Navigate to **Manage Data > Collection** and select the **Open Telemetry Collection** tab. On the top right of the page, click **Add Collector**.<br/>![add-collector-link.png](/img/send-data/add-collector-link.png)
```

Comment:
```suggestion
1. Navigate to **Manage Data > Collection** and select the **OpenTelemetry Collection** tab. On the top right of the page, click **Add Collector**.<br/>![add-collector-link.png](/img/send-data/add-collector-link.png)
```

Reply:
@kimsauce Open Telemetry - two words as per the UI.

Reply:
Oh I see in your screenshot above. I'll have the S3M fix it - this is an error and it appears as one word throughout the rest of the Docs. It's the official industry-wide spelling.